index.html

```diff
@@ -0,0 +1,344 @@
+                    css
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    javaScript
+                </button>
+            </li>
+        </ul>
+        <div class="see-more">
+
+            <button class="text-1">
+                See Project
+            </button>
+
+
+        </div>
+
+    </section>
+
+    <section class="more-info">
+        <div class="image3">
+
+        </div>
+        <div class="projects">
+            <h1 class="post">
+                Multi-Post <br>
+                Stories
+            </h1>
+            <div class="items">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="Stories">
+                    <li>
+                        Back End Dev
+                    </li>
+                    <li>
+                        2015
+                    </li>
+                </ul>
+
+            </div>
+
+        </div>
+        <p class="desc">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="skills">
+            <li>
+                <button class="text">
+                    html
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    css
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    javaScript
+                </button>
+            </li>
+        </ul>
+        <div class="see-more">
+
+            <button class="text-1">
+                See Project
+            </button>
+        </div>
+
+
+        </div>
+
+    </section>
+    <section class="About-myself">
+        <h1 class="intro">
+            About Myself
+        </h1>
+        <p class="about-1">
+            Hello I’m a software developer! I can <br> help you build a product , feature or <br> website Look through some of my work <br>and experience! If you like what you see <br>and have a project you need coded, <br> don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+            <h2>LET'S CONNECT</h2>
+            <div class="icons">
+                <ul class="social-media">
+                    <li>
+                        <a href=""> <i class="fa-brands fa-twitter fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-github fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
+                    </li>
+                </ul>
+
```

Comment:
- You did a great job on the design of  your about section  but kindly make sure you implement those points I list below and make sure you implement them for all card
        - Kindly make sure that your icons have the appropriate color. use the one used in the Figma design it will make your design much to the Figma design :+1: 
        -  Kindly Use the appropriate font size and font family for the text it will make your design look like the Figma design :+1: 

## Figma design

![image](https://user-images.githubusercontent.com/95297251/206457222-d07256b3-baa0-4829-b80e-75db7aa2f3e9.png)

## Your design 

![image](https://user-images.githubusercontent.com/95297251/206457462-f0a0ae12-cf44-432a-9954-47b59af6b76a.png)